docs/dev/framework/models/collections.md

```diff
@@ -33,6 +33,12 @@ For more complex conditions, optionally pass arrays as parameters:
 $pages = PageModel::findBy(['language = ?', 'pid = ?'], ['de', 1]);
 ```
 
+Or you can build any query and include conditions:
+
```

Comment:
```suggestion
```

That sentence is not really necessary, as it's the same principle as the example above :)